system_meta/factories.py

```diff
@@ -0,0 +1,56 @@
+"""Factories for the system_meta app."""
+
+from factory import Faker, SubFactory
+from factory.django import DjangoModelFactory
+
+from system_meta.models import IntegratedSystem, Product
+from unified_ecommerce.factories import InactiveDjangoModelFactory
+
+
+class IntegratedSystemFactory(DjangoModelFactory):
+    """Factory for IntegratedSystem model."""
+
+    class Meta:
+        """Meta options for IntegratedSystemFactory."""
+
+        model = IntegratedSystem
+
+    name = Faker("company")
+    description = Faker("text")
+    api_key = Faker("md5")
+    is_active = None
```

Comment:
Should this  and the factory for the model below be false instead?

Reply:
`django-softdelete` uses None to indicate that the model hasn't been deleted, so went with that.

Reply:
I think maybe we want to go with a name different than `is_active` then - `is_active` as a name implies truthiness, but the actual truthiness is inverted:


```
In [8]: for product in models.Product.all_objects.all():
   ...:     print((product.is_active, bool(product.is_active)))
   ...:
(None, False)
(datetime.datetime(2024, 1, 11, 21, 20, 45, 293708, tzinfo=datetime.timezone.utc), True)
```

Maybe we should just use the default `deleted_at` and then add an `@property` for `is_active`?